nss.yaml

```
@@ -0,0 +1,73 @@
+package:
+  name: nss
+  version: 3.94
+  epoch: 0
+  description: "Network Security Services (NSS) is a set of libraries designed to support cross-platform development of security-enabled client and server applications."
+  copyright:
+    - license: MPL-2.0
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - mercurial
+      - bash
+      - py3-gyp-next
+      - samurai
+      - perl
+      - zlib-dev
+      - libnspr
+      - libnspr-dev
+      - sqlite-dev
+
+var-transforms:
+  - from: ${{package.version}}
+    match: \.
+    replace: _
+    to: nss-package-version
+
+pipeline:
+  # we need to use runs because melange dont have a mercurial checkout pipeline
+  - runs: |
+      hg clone -r NSS_${{vars.nss-package-version}}_RTM https://hg.mozilla.org/projects/nss
+      cd nss
+      ./build.sh -c --disable-tests --opt --system-nspr --system-sqlite
+
+      mkdir -p "${{targets.destdir}}"/usr/lib
+      mkdir -p "${{targets.destdir}}"/usr/bin
+      mkdir -p "${{targets.destdir}}"/usr/include/nss
+
+      mv ../dist/Release/lib/*.so "${{targets.destdir}}"/usr/lib
+      mv ../dist/Release/bin/* "${{targets.destdir}}"/usr/bin
+      mv ../dist/public/nss/* "${{targets.destdir}}"/usr/include/nss
+
+    - uses: strip
```

My suggestion was overly indented 🙃 
```suggestion
  - uses: strip
```